pkg/executor/joiner.go

```diff
@@ -490,6 +499,10 @@ func (naaj *nullAwareAntiSemiJoiner) onMissMatch(_ bool, outer chunk.Row, chk *c
 	chk.AppendRowByColIdxs(outer, naaj.lUsed)
 }
 
+func (naaj *nullAwareAntiSemiJoiner) isSemiJoinWithoutCondition() bool {
+	return len(naaj.conditions) == 0
```

Comment:
Is this optimazation works for nullaware join? And since only `joinMatchedProbeSideRow2Chunk` is check `isSemiJoinWithoutCondition`, nullaware join is not optimized even if it return `true` here.

Reply:
No. This function is useless for null-aware semi join.